aruco_ros/src/aruco_detector.cpp

```diff
@@ -316,45 +327,66 @@ class ArucoSimple
     arucoMsg.error_code = error_condition;
     arucoMsg.error_message = error_message;
     arucoMsg.pose = poseMsg;
+
+    // Include the corner points of the marker, so we an reconstruct its position in the UI
+    for (int i = 0; i < 4; ++i) {
+      aruco_msgs::Corner corner;
+      corner.x = marker[i].x;
```

Comment:
is there any way to confirm that the are four elements in marker?

Reply:
assert()? Marker is just an extension of a cv::point2f vector, so we can just assert the the size of the vec is 4